### DISCUSSION TITLE
Pause button

### DISCUSSION THREAD

**Discussion by Voidheartd** [Erabior/RouteManager#45] [Ideas]
It'd be nice to have some way to pause the route manager since switching to manual and back doesn't resume so well. I often want to stop at the route's endpoints to wait for more passengers, and it'd be nice to not have to confuse the route manager by switching to manual as a way to pause.

**Comment by Voidheartd** [comment]
For now, I will use a fusee to prevent departure.
A pause would also be nice for taking the loco to refuel.

**Reply by AdamC1228** [reply]
I will have to do some testing but i believe that the refactoring and new code I have proposed for v2.0 will improve this. The route manager should be better at picking up where it left off and proceed as if nothing had otherwise happened.

**Reply by AdamC1228** [reply]
Feature implemented.
